ansible_roles/roles/aws_create/tasks/aws_instance_create.yml

```diff
@@ -243,13 +219,3 @@
   when:
    - config_info.cloud_numb_networks != 0
    - config_info.os_vendor == "amazon"
-
```

Comment:
Metadata is now pulled from different location.  We were only getting the metadata for one system when we had 2 for networks.